src/main/java/com/nylas/Event.java

```diff
@@ -149,6 +158,7 @@ public void setRecurrence(Recurrence recurrence) {
 		Maps.putIfNotNull(params, "location", getLocation());
 		Maps.putIfNotNull(params, "participants", getParticipants());
 		Maps.putIfNotNull(params, "busy", getBusy());
+		Maps.putIfNotNull(params, "metadata", getMetadata());
```

Comment:
turns out that the server gives a 500 error when doing a PUT with an empty metadata map.  need to get that fixed or add a special case here to omit empty maps

https://app.clubhouse.io/nylas/story/59971/put-event-with-empty-metadata-results-in-500-error